wiki/Difficulties/osu!/en.md

```diff
@@ -0,0 +1,10 @@
+# osu!
+
+There are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!standard [mapset](/wiki/mapset).
```

Comment:
"There are the five..." sounds odd to me.

`There are five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!standard [mapset](/wiki/mapset).`